src/StoredEventRepository.php

```diff
@@ -17,4 +17,6 @@ public function persist(ShouldBeStored $event, string $uuid = null, int $aggrega
     public function persistMany(array $events, string $uuid = null, int $aggregateVersion = null): LazyCollection;
 
     public function update(StoredEvent $storedEvent): StoredEvent;
+
+    public function getLatestVersion(string $aggregateUuid): int;
```

Comment:
Maybe we should rename this to `->getLatestAggregateVersion` so it's clear to someone implementing the interface that they need to return the aggregate's version

Reply:
Good idea. Done!